### PR TITLE
Citizen movement fix

### PR DESCRIPTION
**Pull request by lovalmidas**
Contains two issue fixes related to citizen movement:
 - Health_min was likely set to sprawl in error.
image

 - Lower-level citizens (starting from Lv 0) were given priority in choosing their cities. This often results in the cities running out of housing for higher-level citizens, resulting in their departure to cities, even those without any Lv 5 jobs. 
image
 
This results in volatile employment and population at these higher levels. Changes to code at two locations to reverse the order of process may help stabilize this.

### CODE REVIEW

File: lib/mayor_game_web/templates/dashboard/show.html.heex [hunk 1]
```diff
@@ -11,54 +11,65 @@
...
+          <span>
+            <%= Number.Delimit.number_to_delimited(length(@cities)) %>
+          </span>
+        </div>
+        <div class="flex-1" />
+        <div class="flex flex-col items-center gap-1 w-24">
+          <span>Population</span>
+          <img src="/images/citizen.svg" alt="city" height="36" width="36" />
+          <span>
+            <%= Number.Delimit.number_to_delimited(@citizen_sum) %>
```
Comment (warronbebster) [inline lib/mayor_game_web/templates/dashboard/show.html.heex:33]: Where is @citizen_sum generated? I would expect this to be somewhere in `dashboard_live.ex`, but don't see any changes in that file in this PR

File: lib/mayor_game_web/templates/dashboard/show.html.heex [hunk 2]
```diff
@@ -114,6 +125,33 @@
           <% end %>
         </div>
       <% end %>
+      
+      <!-- sort options-->
+      <div class="flex flex-row items-center space-x-1 pb-6">
+        <div class="flex-1" />
+        <button
```
Comment (warronbebster) [inline lib/mayor_game_web/templates/dashboard/show.html.heex:132]: Same question here — this is a great idea, but where is `sort_by_name` instantiated?

**Comment by warronbebster** [comment]
Nice! Which file has the changes for starting from level 5 jobs instead of 0? I think I fixed this yesterday but curious how you did it as well

File: lib/mayor_game_web/templates/dashboard/show.html.heex (hunk 1 above)
Comment (lovalmidas) [reply lib/mayor_game_web/templates/dashboard/show.html.heex:33]: They were in dashboard_live.ex but those changes were wiped in a merge commit 6530cb4. Technically this is not related to citizen fix (it belongs to lvm_ui_beautification) but somehow the three branches share the same code.

File: lib/mayor_game_web/templates/dashboard/show.html.heex (hunk 2 above)
Comment (lovalmidas) [reply lib/mayor_game_web/templates/dashboard/show.html.heex:132]: Same as above, they were wiped by a recent merge commit.

**Comment by lovalmidas** [comment]
The changes specifically for citizen movement are in [66ad0db](https://github.com/warronbebster/mayor_game/pull/11/commits/66ad0db66bc3fbf4a0304118c69636a86720cd33)

There are only 2 relevant changes:
 - Reversing 0..5 to 5..0//-1 for Round 1. This resolves priority for higher-level citizens, but only part of the overall movement issue. You have already applied this in your recent patch.
 
 - An explicit reverse operation in the list used for Round 2. This resolves priority for higher-level citizens who have not yet found jobs. This seems to resolve the rest.
 
 No change is needed for Round 3 since it explicitly uses shuffle.
  
 Anyway, I intend to close this issue first, and open a new PR, because (1) I mismanaged what should be separate branches, and (2) these changes are obsoleted by the new update yesterday.

**Pull request closed without merging**